search/src/Components/wrapper/upload.js

```diff
@@ -0,0 +1,145 @@
+import React, {Component} from 'react';
+import UploadDialog from '../upload/upload_dialog';
+import AuthPrompt from '../auth/auth_prompt';
+import { withStyles } from '@material-ui/core/styles';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+import axios from 'axios';
+import Button from '@material-ui/core/Button';
+
+
+const useStyles = (theme) => ({
+    upload_container: {
+        margin: theme.spacing(10)
+    },
+    formControl: {
+        margin: theme.spacing(2),
+        minWidth: 200,
+    },
+})
+
+const baseURL = new URL(window.location.origin);
+const upload_status_mapping = {
+    'N': 'None',
```

Comment:
> Is there a reason we should have codes rather than these full strings as the status in the database? It seems an unnecessary layer of obfuscation.

Do you mean by just having 'N' instead of 'None'?

Reply:
I did commit this as so though I think showing user "Awaiting Review" makes more sense than "AR" especially to new users.

Reply:
I mean that we should definitely show "Awaiting Review" not "AR" in the frontend, but that even in the backend there's little benefit to unintelligible codes when we could use the full words.

Reply:
I think this is the opposite of what you've changed.